### PR TITLE
Default row argument for graph creation, removing debug lines

### PR DESCRIPTION
**Pull request by matbesancon**

### CODE REVIEW

**Comment by matbesancon** [comment]
Fixes #86

**Comment by matbesancon** [comment]
also @ChrisRackauckas I don't have the rights on the repo anymore with the change of org

**Comment by matbesancon** [comment]
https://travis-ci.org/github/JuliaDiff/SparseDiffTools.jl/builds/669186261
The plot thickens, an error with 1.3 on mac only

**Comment by matbesancon** [comment]
ok finally passing. @ChrisRackauckas good to merge here

File: src/coloring/matrix2graph.jl [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-        matrix2graph(sparse_matrix)
+        matrix2graph(sparse_matrix, [partition_by_rows::Bool=true])
 
 A utility function to generate a graph from input
 sparse matrix, columns are represented with vertices
 and 2 vertices are connected with an edge only if
 the two columns are mutually orthogonal.
 """
-function matrix2graph(sparse_matrix::SparseMatrixCSC{<:Number, Int}, partition_by_rows::Bool)
+function matrix2graph(sparse_matrix::SparseMatrixCSC{<:Number, Int}, partition_by_rows::Bool=true)
```
Comment (ChrisRackauckas) [inline src/coloring/matrix2graph.jl:9]: why default to rows?

**Comment by matbesancon** [comment]
It was the default before adding the option

**Comment by ChrisRackauckas** [comment]
oh yes, partitioning by rows is Forward mode.

**Pull request merged**